packages/e2fsprogs/mke2fs.conf

```diff
@@ -0,0 +1,51 @@
+# We store the shipped configuration in the repository rather than using the
+# latest upstream default. New features must be added with care, as they may
+# not be compatible with the running kernel after a downgrade.
```

Comment:
```suggestion
# Note: new features must be added with care, as they may not be compatible
# with the running kernel after a downgrade.  To make changes more obvious, we
# store the shipped configuration in the repository rather than using the
# latest upstream default.
```
This is definitely a nit, but reordering this a bit might take the sting away from referring to the repo in a file in the image.  We could also just remove the sentence about the repo, since the critical part is the warning.

Reply:
I've removed the repository sentence.